src/Models/ArduinoDeviceBase.ts

```diff
@@ -194,7 +189,7 @@ export abstract class ArduinoDeviceBase implements Device {
   }
 
   async generateSketchFile(
-      sketchTemplateFileName: string, board: Board, boardInfo: string,
+      fileInfo: TemplateFileInfo, board: Board, boardInfo: string,

```

Comment:
Modify this function using FileUtility function instead of fs-plus file operation function. Or the creation of file will fail when create file from remote side.

Reply:
Can see the `containerization` branch for detail.